src/Fantomas.Tests/UnionTests.fs

```diff
@@ -618,7 +618,7 @@ namespace X
 type UnresolvedAssemblyReference = UnresolvedAssemblyReference of string * AssemblyReference list
 
 type ResolvedExtensionReference =
-    ResolvedExtensionReference of string * AssemblyReference list * Tainted<ITypeProvider> list
+    | ResolvedExtensionReference of string * AssemblyReference list * Tainted<ITypeProvider> list
```

Comment:
I thought this was going to be a new mode, and shouldn't affect existing code? (I don't mind, certainly - I just would expect a bit of pushback from other people.)